src/data/models/course.ts

```diff
@@ -45,6 +46,7 @@ export type CourseModelParams = {
     dateCreated: string;
     guid: string;
   };
+  language?: Maybe<string>;
```

Comment:
the echo content package database supports a `language` string field, which is not used by any packages at the moment. Using this field for the default foreign language simplifies the server changes